source/issues-and-help.txt

```diff
@@ -4,4 +4,51 @@ Issues & Help
 
 .. default-domain:: mongodb
 
-Lorem ipsum
+We are lucky to have a vibrant MongoDB Java community with lots of varying
+experience of using the Java driver. We often find the quickest way to get
```

Comment:
```suggestion
experiences of using the Java driver. We often find the quickest way to get
```

Something about this first sentence just doesn't seem right to me. Open to this change or a reword.